controllers/vehicles.js

```diff
@@ -0,0 +1,129 @@
+/* ******************************
+ * Required Resources
+ * ******************************/
+const mongodb = require('../config/db');
+const ObjectId = require('mongodb').ObjectId;
+
+/* ******************************
+ * Get all vehicles
+ * ******************************/
+const getAll = async (req, res) => {
+  const result = await mongodb.getDatabase().db().collection('vehicles').find();
```

Comment:
I see you used vehicles and I think it should be vehicle (without the 's' plural) this would help to keep consistency

Reply:
I changed all instances of vehicles to vehicle